sailfishusermanager.cpp

```diff
@@ -397,18 +407,18 @@ void SailfishUserManager::removeUser(uint uid)
     m_exitTimer->start();
 
     if (!removeHome(uid)) {
-        sendErrorReply(QStringLiteral(SailfishUserManagerErrorHomeRemoveFailed), QStringLiteral("Removing user home failed"));
-        return;
```

Comment:
Why was this return removed? Also this seems like it could be `if (!removeHome(uid) && calledFromDBus()) { /* ... */ }`

Reply:
Removed because it prevented removing user without existing home, like "/tmp/guest_home" gets deleted in reboot.

Reply:
Shouldn't it still return if that failure happens and the user removed is not guest user?

Reply:
Anyway it feels odd if it emits an error but continues possibly succeeding and emitting alse the success case.